newIDE/app/src/Utils/ColorTransformer.js

```diff
@@ -11,3 +12,16 @@ export const rgbToHex = (r: number, g: number, b: number) =>
  */
 export const rgbToHexNumber = (r: number, g: number, b: number) =>
   (r << 16) + (g << 8) + b;
+
+/**
+ * Convert a hex color value to an rgb object value.
+ */
+export const getRGBColorFromHex = (hex: string) => {
+  const rgb = PIXI.utils.hex2rgb(`0x${hex.replace('#', '')}`);
```

Comment:
Let's not add a dependency on Pixi for Utils - almost all the interface should be agnostic about Pixi, just the InstancesRenderer (i.e: the scene editor) and the RenderedInstances (the classes used to display instances) should use Pixi.

The method is very simple and we can copy it here: https://pixijs.download/v5.0.0-alpha.2/docs/packages_utils_src_color_hex.js.html#line1

Reply:
Good thinking, I will copy it to my utility method and remove the import :)

Reply:
Note that hex2rgb was expecting a number. To get this working properly and safely with Flow, you should parse the hex string as a number:

`const hexColor = parseInt(hex.replace('#', ''), 16); // The hex color as a number`

then you can do what is done in the method:

`const red = ((hex >> 16) & 0xFF); // I've removed the /255 as you might not need it` 

(do the same for green and blue and you can return them as is I think - not sure `floor`ing is needed at all)

Finally, let's name the method as `hexToRGBColor` to be slightly more similar to other methods in the file.

Reply:
worked like a charm! Thank you @4ian :D